plugins/@grouparoo/mailchimp/src/lib/shared/updateProfile.ts

```diff
@@ -73,26 +73,23 @@ export const updateProfile: UpdateProfileMethod = async ({
     }
   }
 
-  try {
-    const userResponse = await getUser(client, listId, mailchimpId);
-    exists = userResponse !== null;
+  const userResponse = await getUser(client, listId, mailchimpId);
+  exists = userResponse !== null;
```

Comment:
I'm not sure this `exists` flag is serving us anymore that we aren't catching. can probably just use
> const existingUser = await getUser(client, listId, mailchimpId)

Reply:
and `if(existingUser)`